blocklist-client/README.md

```diff
@@ -14,12 +14,29 @@ You can run the blocklist client by providing the required environment variables
 #### Required Environment Variables
 - BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL=`<provider-url>`
 - BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY=`<your_api_key>`
+- BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS=`<true|false>`
+
+Currently we implemented two risk providers, which can be chosen via the `BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS` environment variable. If set to `true`, the [sanctions](https://public.chainalysis.com/docs/index.html#introduction) provider will be used, otherwise the risk provider will be used. We currently lost information about the risk provider, so only with the `BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS` set to `true` the client will work.
+
+   ```bash
+    BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL=https://your-risk-provider-api.com/ BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY=your_api_key BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS=true ../target/release/blocklist-client 
+   ```
 
 #### Optional Environment Variables
 If not specified, the default values from `./src/config/default.toml` will be used.
 - BLOCKLIST_CLIENT_SERVER__HOST=`<server-hostname-or-ip>`
 - BLOCKLIST_CLIENT_SERVER__PORT=`<server-port>`
 
    ```bash
-    BLOCKLIST_CLIENT_SERVER__HOST=127.0.0.1 BLOCKLIST_CLIENT_SERVER__PORT=8080 BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL=https://your-risk-provider-api.com/ BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY=your_api_key  ../target/release/blocklist-client 
+    BLOCKLIST_CLIENT_SERVER__HOST=127.0.0.1 BLOCKLIST_CLIENT_SERVER__PORT=8080 BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL=https://your-risk-provider-api.com/ BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY=your_api_key BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS=true  ../target/release/blocklist-client 
```

Comment:
Can you make this default to `true` in code?

Reply:
Thank you!

Reply:
Yes I would make this the default. Please take a look also at the other comments to this PR.